### DISCUSSION TITLE
How to pass secrets into vm

### DISCUSSION THREAD

**Discussion by reegnz** [lima-vm/lima#1068] [Q&A] [answered]
I'm trying to run an Agent in a lima vm (github actions runner to be precise), and this agent uses an API token to authenticate with a service (github.com).

What's the best suggested way of passing in the API Token into the vm without hardcoding it in the instance config?

**Comment by AkihiroSuda** [comment, marked as answer]
The host home directory is visible to the guest, so you can just pass the secret file from there.
